src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -49,25 +50,37 @@ interface Props {
   onClose: () => void
   eventPrefix: string
 }
+export const WRONG_ORDER_MESSAGE = 'Stop Time must be after the start time'
+export const END_TIME_IN_FUTURE_MESSAGE = 'Stop Time cannot be in the future'
+export const TIMES_ARE_SAME_MESSAGE = 'Stop Time must be after the start time'
+export const START_TIME_IN_FUTURE_MESSAGE = 'Start Time cannot be in the future'
 
 export const AnnotationForm: FC<Props> = (props: Props) => {
   const [startTime, setStartTime] = useState(props.startTime)
   const [endTime, setEndTime] = useState(props.endTime)
   const [summary, setSummary] = useState(props.summary)
   const [annotationType, setAnnotationType] = useState(props.type)
 
+  // in the time by itself valid ?(is the format valid)
+  // since we start with default times via the mouse, the formats are valid at first
+  // (until the user edits it)
+  const [endTimeFormatValid, setEndTimeFormatValid] = useState(true)
+  const [startTimeFormatValid, setStartTimeFormatValid] = useState(true)
+
   const dispatch = useDispatch()
 
   const isValidAnnotationForm = (): boolean => {
-    const isValidPointAnnotation = Boolean(summary?.length && startTime)
+    const isValidPointAnnotation = Boolean(
+      summary?.length && startTime && startTimeFormatValid && isStartTimeValid()
```

Comment:
same question here about wrapping this in a `Boolean`

Reply:
see above; fixed now